UnityProject/Assets/Scripts/Core/Editor/MiscFunctions_RRT.cs

```diff
@@ -369,6 +369,10 @@ class LoginBody
 		public static void Generate()
 		{
 
+			var coolsdsa = AllowedReflection.GetTypeByName("LightSource");
+
+			return;
```

Comment:
leave a comment saying that this is disabled

Reply:
is dev function >UnityProject/Assets/Scripts/Core/Editor/MiscFunctions_RRT.cs